resource-managers/cook/pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>twosigma</groupId>
       <artifactId>cook-jobclient</artifactId>
-      <version>0.2.0-SNAPSHOT</version>
+      <version>0.2.0</version>
```

Comment:
Is the 0.2.0 still using http 4.3.6? Can you show me the artifact link

Reply:
We reviewed this together but for future reference: https://clojars.org/twosigma/cook-jobclient/versions/0.2.0